### PR TITLE
Cleanup properties -- don't escape spaces

### PR DESCRIPTION
**Pull request by jsoref**
This is for discussion.

Notes:
* Items were identified by IntelliJ
* Individual commits are mostly self contained
* Some commits are probably real bug fixes

### Proposed changelog entries

* l10n: don't escape spaces 
-- something should go here, as it actually has potential to change the user experience...

### Submitter checklist

- [ ] JIRA issue is well described
- [ ] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [x] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### CODE REVIEW

**Review by jvz — APPROVED** [review]

File: core/src/main/resources/hudson/diagnosis/HudsonHomeDiskUsageMonitor/index_da.properties [hunk 1]
```diff
@@ -20,12 +20,12 @@
...
 
-description.1=\ 
+description.1=\
 Dit <tt>JENKINS_HOME</tt> ({0}) er n\u00e6sten fyldt. \
 N\u00e5r dette direktorie fylder helt op vil det sprede d\u00f8d og \u00f8del\u00e6ggelse, da Jenkins ikke vil kunne gemme mere data.
-solution.2=\ 
+solution.2=\
 Flyt <tt>JENKINS_HOME</tt> til et st\u00f8rre diskafsnit.
-JENKINS_HOME\ is\ almost\ full=JENKINS_HOME\ er\ n\u00e6sten\ fyldt\ op
+JENKINS_HOME\ is\ almost\ full=JENKINS_HOME\ er\ n\u00e6sten\ fyldt op
```
Comment (MRamonLeon) [inline core/src/main/resources/hudson/diagnosis/HudsonHomeDiskUsageMonitor/index_da.properties]: Shouldn't we remove the  \ at the right before the blanks?

File: core/src/main/resources/hudson/slaves/JNLPLauncher/help_bg.properties [hunk 2]
```diff
@@ -20,7 +20,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-# Allows an agent to be launched using <a href="https://en.wikipedia.org/wiki/Java_Web_Start" target="_blank">Java Web Start</a>.<br>\ 
+# Allows an agent to be launched using <a href="https://en.wikipedia.org/wiki/Java_Web_Start" target="_blank">Java Web Start</a>.<br> \
```
Comment (MRamonLeon) [inline core/src/main/resources/hudson/slaves/JNLPLauncher/help_bg.properties]: I think we can remove all the \ of these comments, like this:

```suggestion
# Allows an agent to be launched using <a href="https://en.wikipedia.org/wiki/Java_Web_Start" target="_blank">Java Web Start</a>.<br>
```

File: core/src/main/resources/hudson/tasks/Maven/help_bg.properties [hunk 3]
```diff
@@ -20,7 +20,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-# Jenkins passes <a href="{0}/env-vars.html" target="_new">various environment \ 
+# Jenkins passes <a href="{0}/env-vars.html" target="_new">various environment \
```
Comment (MRamonLeon) [inline core/src/main/resources/hudson/tasks/Maven/help_bg.properties]:
```suggestion
# Jenkins passes <a href="{0}/env-vars.html" target="_new">various environment
```

File: core/src/main/resources/hudson/triggers/SCMTrigger/AdministrativeMonitorImpl/message_bg.properties [hunk 4]
```diff
@@ -22,7 +22,7 @@
 
 # There are more SCM polling activities scheduled than handled, so \ 
 # the threads are not keeping up with the demands. \ 
-# <a href='descriptor/hudson.triggers.SCMTrigger/'>Check if your polling is \ 
+# <a href='descriptor/hudson.triggers.SCMTrigger/'>Check if your polling is \
```
Comment (MRamonLeon) [inline core/src/main/resources/hudson/triggers/SCMTrigger/AdministrativeMonitorImpl/message_bg.properties]:
```suggestion
# <a href='descriptor/hudson.triggers.SCMTrigger/'>Check if your polling is
```

File: core/src/main/resources/hudson/triggers/SCMTrigger/AdministrativeMonitorImpl/message_pt_BR.properties [hunk 5]
```diff
@@ -22,7 +22,7 @@
 
 # There are more SCM polling activities scheduled than handled, so \ 
 # the threads are not keeping up with the demands. \ 
-# <a href='descriptor/hudson.triggers.SCMTrigger/'>Check if your polling is \ 
+# <a href='descriptor/hudson.triggers.SCMTrigger/'>Check if your polling is \
```
Comment (MRamonLeon) [inline core/src/main/resources/hudson/triggers/SCMTrigger/AdministrativeMonitorImpl/message_pt_BR.properties]:
```suggestion
# <a href='descriptor/hudson.triggers.SCMTrigger/'>Check if your polling is
```

File: core/src/main/resources/hudson/util/InsufficientPermissionDetected/index_pt_BR.properties [hunk 6]
```diff
@@ -34,7 +34,7 @@ de seguran\u00e7a, poder\u00e1 desativ\u00e1-lo.
 Error=Erro
 # \ 
 # For how to turn off security manager in your container, refer to \ 
-# <a href="https://jenkins.io/redirect/troubleshooting/disable-security-manager"> \ 
+# <a href="https://jenkins.io/redirect/troubleshooting/disable-security-manager"> \
```
Comment (MRamonLeon) [inline core/src/main/resources/hudson/util/InsufficientPermissionDetected/index_pt_BR.properties]:
```suggestion
# <a href="https://jenkins.io/redirect/troubleshooting/disable-security-manager">
```

File: core/src/main/resources/jenkins/model/Jenkins/oops_pl.properties [hunk 7]
```diff
@@ -19,7 +19,7 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
-Twitter\:\ @jenkinsci=Twitter\:\ @jenkinsci
+Twitter\:\ @jenkinsci=Twitter\: @jenkinsci
```
Comment (MRamonLeon) [inline core/src/main/resources/jenkins/model/Jenkins/oops_pl.properties]: Don't sure if we need the \ before the :

**Review by MRamonLeon — COMMENTED** [review]
Some suggestions

File: core/src/main/resources/hudson/slaves/JNLPLauncher/help_bg.properties (hunk 2 above)
Comment (jsoref) [reply core/src/main/resources/hudson/slaves/JNLPLauncher/help_bg.properties]: Technically they're comments, so nothing matters. But the spirit is such that I think they want to belong, and I don't want to make a sweeping change.

**Review by oleg-nenashev — APPROVED** [review]
will merge tomorrow if no negative feedback

**Comment by oleg-nenashev** [comment]
@MRamonLeon are you fine with merging?

**Review by MRamonLeon — APPROVED** [review]
I'm fine, the only one was the Twitter value and it's fine as it is.

**Pull request merged**